src/test/perf_counters.cc

```diff
@@ -255,3 +257,369 @@ TEST(PerfCounters, read_avg) {
   t2.join();
   t1.join();
 }
+
+static PerfCounters* setup_test_perfcounter4(std::string name, CephContext *cct)
+{
+  PerfCountersBuilder bld(cct, name,
+	  TEST_PERFCOUNTERS2_ELEMENT_FIRST, TEST_PERFCOUNTERS2_ELEMENT_LAST);
+  bld.add_u64(TEST_PERFCOUNTERS2_ELEMENT_FOO, "foo");
+  bld.add_time(TEST_PERFCOUNTERS2_ELEMENT_BAR, "bar");
+
+  PerfCounters* counters = bld.create_perf_counters();
+  cct->get_perfcounters_collection()->add(counters);
+  return counters;
+}
+
+TEST(PerfCounters, TestLabeledCountersOnly) {
+  std::string_view EMPTY_DUMP_FORMAT_RAW = R"({}
```

Comment:
@cbodley @idryomov I realized that this constant was only being used for this test so I moved it into the test. Should I still have the name be in all caps? I also wasn't sure if `std::string_view` was the appropriate type to use as for a non constexpr string constant.

Reply:
string_view is good. 👍 since the variable isn't modified in the test, you should make it const or constexpr

string_view stores a length, so its constructor has to call `strlen()` on the input string. the benefit to using constexpr is that this can happen at compile-time instead of runtime

Reply:
> I realized that this constant was only being used for this test so I moved it into the test

Honestly, since it's used just in two place and is basically an equivalent of an empty string, I'd have open-coded it.